pages/es/lb4/index.md

```diff
@@ -0,0 +1,51 @@
+---
+lang: es
+title: LoopBack 4
+toc: false
+keywords: LoopBack 4
+sidebar: lb4_sidebar_es
```

Comment:
Sorry, promise this is the last thing, but can you update this to be `es_lb4_sidebar`? (somehow missed it before) Then we'll land it.